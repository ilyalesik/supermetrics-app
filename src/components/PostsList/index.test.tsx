import React from "react";
import renderer from "react-test-renderer";
import { PostsList } from "./index";

it("renders PostsList", () => {
  const tree = renderer
    .create(
      <PostsList
        items={[
          {
            created_time: "2022-03-28T19:06:39+00:00",
            from_id: "user_1",
            from_name: "Filomena Cort",
            id: "post624236da322ce_86ec7f87",
            message:
              "protection racism mood contrary pavement drama sample manufacture printer",
            type: "status",
          },
          {
            created_time: "2022-03-28T13:32:35+00:00",
            from_id: "user_1",
            from_name: "Filomena Cort",
            id: "post624236da322d9_dbc2eea1",
            message:
              "plane falsify rear admit difficulty dorm core thin flat wrestle reputation beer syndrome reliance mail tumour intermediate speech recommendation linen combine move mug fling facade wrist wagon buy permission test leave dynamic expertise",
            type: "status",
          },
        ]}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
