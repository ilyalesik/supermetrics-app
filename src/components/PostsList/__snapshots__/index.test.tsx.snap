// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PostsList 1`] = `
<ul
  className="posts-lists"
>
  <li
    className="posts-list-item"
  >
    <div
      className="posts-list-item-header"
    >
      Mon Mar 28 2022 22:06:39 GMT+0300 (Moscow Standard Time)
    </div>
    <div
      className="posts-list-item-message"
    >
      protection racism mood contrary pavement drama sample manufacture printer
    </div>
  </li>
  <li
    className="posts-list-item"
  >
    <div
      className="posts-list-item-header"
    >
      Mon Mar 28 2022 16:32:35 GMT+0300 (Moscow Standard Time)
    </div>
    <div
      className="posts-list-item-message"
    >
      plane falsify rear admit difficulty dorm core thin flat wrestle reputation beer syndrome reliance mail tumour intermediate speech recommendation linen combine move mug fling facade wrist wagon buy permission test leave dynamic expertise
    </div>
  </li>
</ul>
`;
